kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/release/ReleaseIdComparator.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.server.services.impl.release;
+
+import static org.kie.server.services.impl.release.ComparableReleaseId.adapt;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+import org.kie.api.builder.ReleaseId;
+
+/**
+ * Compares release ids.
+ */
+public class ReleaseIdComparator implements Comparator<ReleaseId> {
```

Comment:
I'd prefer to put this class in kie-api next to ReleaseId itself rather in kie server only